packages/@aws-cdk/aws-ecr/lib/auth-token.ts

```diff
@@ -1,7 +1,9 @@
 import * as iam from '@aws-cdk/aws-iam';
 
 /**
- * Authorization token to access ECR repositories via Docker CLI.
+ * Authorization token to access private ECR repositories in the current environment via Docker CLI.
```

Comment:
It's worth mentioning that retrieving the token in the private case doesn't mean the principal will be able to pull images (this is controlled by the resource policy on the repository).